src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -536,7 +536,13 @@ namespace {
     }
 
     void Schedule::handleNUPCOL( const DeckKeyword& keyword, size_t currentStep) {
-        const int nupcol = keyword.getRecord(0).getItem("NUM_ITER").get<int>(0);
+        int nupcol = keyword.getRecord(0).getItem("NUM_ITER").get<int>(0);
+        if (keyword.getRecord(0).getItem("NUM_ITER").defaultApplied(0)) {
```

Comment:
I really think the default value should be updated in the Json file; possibly you can add a `NUPCOL` keyword with default value 12 here: https://github.com/OPM/opm-common/tree/b8605532f4d66dcb9c33476940622c4b62213588/src/opm/parser/eclipse/share/keywords/900_OPM -  then that will override the Eclipse100 version.

Reply:
I will change the default in the Json file and add a comment there + keep the note if NUPCOL is used with 1*.